src/KonnektingDevice.h

```diff
@@ -161,14 +161,16 @@ class KonnektingDevice {
 
     bool isReadyForApplication();
 
+    void setProgState(bool state);
+
     int getFreeEepromOffset();
 
 private:
 
     bool _rebootRequired = false;
     bool _initialized = false;
 
-    int _progbtnCount = 0;
+    byte _progbtnCount = 0;
```

Comment:
maybe add ifdef for REBOOT_BUTTON ?? That way it's not in  memory if it's not used at all.

Reply:
#ifdef added